pkg/controller/jobs/job/job_controller_test.go

```diff
@@ -2397,12 +2400,16 @@ func TestReconciler(t *testing.T) {
 				Key:      "tolerationkey2",
 				Operator: corev1.TolerationOpExists,
 				Effect:   corev1.TaintEffectNoSchedule,
-			}).Suspend(false).Obj(),
+			}).NodeSelector("node-label", "value").
```

Comment:
to prove that the node selectors have no impact, the selectors in the job should be different then the ones in the workload.